manifests/fact.pp

```diff
@@ -20,7 +20,6 @@
       owner   => $facter::facts_file_owner,
       group   => $facter::facts_file_group,
       mode    => $facter::facts_file_mode,
-      require => File['facts_d_directory'],
```

Comment:
this introduced a style issue

https://travis-ci.org/ghoneycutt/puppet-module-facter/jobs/295429401#L650